spec/controllers/spree/admin/base_controller_spec.rb

```diff
@@ -24,7 +24,7 @@ def index
 
         it "passes a prefix to the serializer method and renders with serializer" do
           expect(controller).to receive(:serializer).with(prefix) { "SerializerClass" }
-          expect(controller).to receive(:render).with(json: data, serializer: "SerializerClass")
+          expect(controller).to receive(:render).with({ json: data, serializer: "SerializerClass" })
```

Comment:
Interesting, I would have thought that Rails would have changed the controller arguments over to keyword arguments, but I guess they didn't.